app/views/contexts/summary/question.py

```diff
@@ -22,6 +22,14 @@ def _get_answer(self, answer_store, answer_id):
         if answer:
             if isinstance(answer.value, str):
                 return escape(answer.value)
+            elif isinstance(answer.value, dict):
+                answer_dict = {}
+                for answer_field in answer.value:
+                    if isinstance(answer.value[answer_field], str):
+                        answer_dict[answer_field] = escape(answer.value[answer_field])
+                    else:
+                        answer_dict[answer_field] = answer.value[answer_field]
+                return answer_dict
```

Comment:
The dictionary `items` method can be used to make this a little clearer:
```suggestion
                escaped_answer = {}
                for key, value in answer.value.items():
                    escaped_answer[key] = escape(value) if isinstance(value, str) else value
                return escaped_answer
```

Reply:
Done, thanks!